Description: Reverse the patch merged upstream which swallows actions in
             notifications.  It is more appropriate for applications to make
             that decision rather than a library.
From: David Barth <david.barth@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/349550
Index: notify-sharp/src/Notification.cs
===================================================================
--- notify-sharp.orig/src/Notification.cs	2011-02-26 02:13:37.452598287 +0800
+++ notify-sharp/src/Notification.cs	2011-02-26 02:14:10.562409163 +0800
@@ -341,13 +341,10 @@
 		}
 
 		public void AddAction (string action, string label, ActionHandler handler) {
-			if (Notifications.Global.Capabilities != null &&
-			    Array.IndexOf (Notifications.Global.Capabilities, "actions") > -1) {
-				lock (action_map) {
-					action_map[action] = new ActionTuple (label, handler);
-				}
-				Update ();
+			lock (action_map) {
+				action_map[action] = new ActionTuple (label, handler);
 			}
+			Update ();
 		}
 
 		public void RemoveAction (string action) {
